API/projectClass/contrastsClass.m

```diff
@@ -71,7 +71,7 @@
             contrastBackgroundActions = cell(1,nContrasts);
             contrastBulkIns = ones(1,nContrasts);
             contrastBulkOuts = ones(1,nContrasts);
-            contrastDomainRatios = zeros(1,nContrasts);
+            contrastDomainRatios = ones(1,nContrasts);
```

Comment:
I am not sure why this is changed?

Reply:
Just for consistency with the rest of the arrays and tests now that each entry is filled in or set to -1

Reply:
A few lines further down, we loop through each contrast and set the value if the contrast has a domain ratio, or set it to -1. I therefore can't see any issue with this change. Having said that, I take your point that there is potential for problems with this default, so I'll change back. I'm inclined to think, especially with the check indices routine now present, that setting all of these to a default of zero is the better option. One to talk over on Monday maybe?